admin/test/javascripts/fixtures/existing_navigation_taxon.html.haml

```diff
@@ -0,0 +1,10 @@
+.new-navigation-link__section{ data: { new_navigation_taxon: 'bar' } }
```

Comment:
this is a copy of `new_navigation_taxon.html.haml`, except it mocks out an edit page where the `navigable_id` has already been selected.